content/applications/hr/fleet/configuration.rst

```diff
@@ -0,0 +1,234 @@
+  in the :guilabel:`Engine` section of the vehicle form. Modifying the :guilabel:`CO2 Emissions`
+  field will adjust the value in the :guilabel:`CO2 fee` field.
+- :guilabel:`Cost (Depreciated)`: Enter the monthly cost for the vehicle, which appears in the
+  salary configurator. This value impacts the gross and net salary of the employee who is assigned
+  to the vehicle. This figure is depreciated over time.
+- :guilabel:`Total Cost (Depreciated))`: This value is the :guilabel:`Cost (Depreciated)` and the
+  :guilabel:`C02 fee` fields combined, and also is depreciated over time.
+
+.. note::
+   The :guilabel:`Cost (Depreciated)` does not depreciate automatically on the vehicle model, it
+   only depreciates based on the contract linked to a specific vehicle and not on the general model.
+
+Engine
+******
+
+- :guilabel:`Fuel Type`: Select the type of fuel the vehicle uses form the drop-down menu. The
+  options are :guilabel:`Diesel`, :guilabel:`Gasoline`, :guilabel:`Hybrid Diesel`, :guilabel:`Hybrid
+  Gasoline`, :guilabel:`Plug-in Hybrid Diesel`, :guilabel:`Plug-in Hybrid Gasoline`,
+  :guilabel:`CNG`, :guilabel:`LPG`, :guilabel:`Hydrogen`, or :guilabel:`Electric`.
+- :guilabel:`CO2 Emissions`: Enter the carbon dioxide emissions the vehicle produces in grams per
+  kilometer (g/km).
+- :guilabel:`CO2 Standard`: Enter the standard amount of carbon dioxide for a similar sized vehicle.
+- :guilabel:`Transmission`: Select the transmission type from the drop-down menu, either
+  :guilabel:`Manual` or :guilabel:`Automatic`.
+- :guilabel:`Power`: If the vehicle is electric or hybrid, enter the power the vehicle uses in
+  kilowatts (kW).
+- :guilabel:`Horsepower`: Enter the vehicle's horsepower in this field.
+- :guilabel:`Horsepower Taxation`: Enter the amount that is taxed based on the size of the vehicles
+  engine. This is determined by local taxes and regulations, and varies depending on the location.
+  It is recommended to check with the accounting department to ensure this value is correct.
+- :guilabel:`Tax Deduction`: This field auto-populates according to the engine specifications, and
+  cannot be modified. The percentage is based on the localization settings and local tax laws.
+
+Vendors tab
+~~~~~~~~~~~
+
+Vendors the vehicle can be purchased from may be added in this tab. To add a vendor, click
+:guilabel:`Add`, and a pop-up loads with a list of all the vendors currently in the database. Select
+the vendor to add by clicking the checkbox next to the vendor name, then click :guilabel:`Select`.
+There is no limit to the number of vendors that can be added to this list.
+
+If a vendor is not in the database, add a vendor by clicking :guilabel:`Create`. A vendor form will
+load. Enter the information in the vendor tab, then click :guilabel:`Save & Close` to add the vendor
+and close the window, or :guilabel:`Save & New` to add the current vendor and create another new
+vendor.
+
```

Comment:
where do I go to see a list of all my vendors in fleet?

Reply:
There isn't a separate list of vehicle vendors- they are the same vendors that you would use in the purchase app. So when you click ADD you see all the vendors in the database, from Azure Interior to BMW of Buffalo. I added a sentence to clarify this- since there is no dedicated vehicle vendor list.